### ISSUE TITLE
Feature: Schrödinger's Cube Pair Detection

### ISSUE THREAD

**Issue by ErinRoseWebs** [ChaosInitiative/Portal-2-Community-Edition#794]
### Which component should be improved?

Gameplay

### Describe your suggestion

Right, so I mentioned this in the QNA stream, but I thought I'd add it here as well for those who weren't there or don't remember. While discussing the Schrödinger's Cube part of the campaign, one of the issues brought up was "if there's multiple pairs, how do you know what cubes connect to which?". Well, the first idea would obviously be "oh, change the color!" However, I don't think that would work. What if someone is colorblind? It's an issue brought up in BEE, hence the idea of using symbols for the cubes. But with these cubes, you can't do that. This is why I also suggested the ability to change light bridge, funnel, and laser textures. That doesn't work as well here. SO:

When holding a Schrödinger's Cube, and if the partner is spawned, the partner cube will highlight when in view, and will have a box outline similar to when you view a portal from behind a brush. This outline can also include an arrow to help the player know which way the cube is facing, similar to how you can use the gradients of the portal outlines to know the portal's orientation. 

### Expected result

Tl;dr, this adds an outline to the partner of a Schrödinger's Cube when picked up and the partner cube is behind a brush, and highlights the partner when in view. Helps player know what cube's are paired together and potentially the orientation of the partner cube.

**Comment by TeamSpen210** [comment]
If an outline seems insufficiently diegetic, perhaps instead the Schrödinger Cube could have some blue/orange lights added to the model (if it doesn't have em already), then when held the partner would blink/switch to orange.

**Comment by ErinRoseWebs** [comment]
oooh yeah thats good as well.
